src/sentry/south_migrations/0344_add_index_ProjectPlatform_last_seen.py

```diff
@@ -0,0 +1,3245 @@
+# -*- coding: utf-8 -*-
+from south.utils import datetime_utils as datetime
+from south.db import db
+from south.v2 import SchemaMigration
+from django.db import models
+
+
+class Migration(SchemaMigration):
+    def forwards(self, orm):
+        # Adding index on 'ProjectPlatform', fields ['last_seen']
+        db.create_index('sentry_projectplatform', ['last_seen'])
```

Comment:
Needs to be created concurrently. Search for `CREATE INDEX CONCURRENTLY` in other migrations.